addons/packages/packetfence.spec

```diff
@@ -583,6 +583,10 @@ if [ ! -f /usr/local/pf/conf/ssl/server.crt ]; then
     cat /usr/local/pf/conf/ssl/server.crt /usr/local/pf/conf/ssl/server.key > /usr/local/pf/conf/ssl/server.pem
 fi
 
+# Create OMAPI key
+if [ ! -f /usr/local/pf/conf/pf_omapi_key ]; then
+    echo $(date) $HOSTNAME | base64 > /usr/local/pf/conf/pf_omapi_key
```

Comment:
Can we use something a little bit random.
Because if i want to crash the dhcpd service i just have do a brute force attack with a hash of a date and the hostname (it can be found).